### PR TITLE
fix: stop carousel from scrolling with few events (#397)

### PR DESCRIPTION
**Pull request by toffee-k21**
**Description**
Fixed issue #397 
Stopped the carousel from scrolling with a few events

### CODE REVIEW

**Review by AceTheCreator — CHANGES_REQUESTED** [review]
@toffee-k21 based on the preview of this PR, the venues should be centered https://deploy-preview-405--peaceful-ramanujan-288045.netlify.app/
AsyncAPI-Conference-10-07-2024_07_39_PM

**Comment by toffee-k21** [comment]
@AceTheCreator I have fixed the issue you mentioned, thanks for giving me the feedback

File: components/Header/header.js [hunk 1]
```diff
@@ -43,11 +45,17 @@ function Header() {
 				</div>
 			</div>
 			<div className='mt-24'>
-				<ReactSlider>
+				{cities.length > 4 || isMobile ? <ReactSlider>
```
Comment (AceTheCreator) [inline components/Header/header.js:48]: Instead of adding the logic in the header section, why not implement it in the react-slider component?

**Review by AceTheCreator — CHANGES_REQUESTED** [review]
Add the logic inside this component inside

File: components/Header/header.js (hunk 1 above)
Comment (toffee-k21) [reply components/Header/header.js]: @AceTheCreator, I added the logic to the react-slider component. Please check if everything looks correct.

**Review by AceTheCreator — CHANGES_REQUESTED** [review]
@toffee-k21 one more thing, make the venue images .webp

**Comment by ashmit-coder** [comment]
> > @toffee-k21 one more thing, make the venue images .webp
> 
> @AceTheCreator .gif when converted to .webp will be static, are you sure to make this change ?

Hey @toffee-k21 the `online edition` image in jpeg format just convert it to .webp and leave the rest to GIFs

**Comment by toffee-k21** [comment]
@AceTheCreator @ashmit-coder Is it fine now?

**Comment by AceTheCreator** [comment]
@toffee-k21 some of the checks are failing

**Review by AceTheCreator — APPROVED** [review]
LGTM 🚀

**Pull request merged**